README.md

```diff
@@ -1 +1 @@
-# Code-review
\ No newline at end of file
+# Alex---Robert
```

Comment:
Please make readme file more descriptive. I think it will be good to write what have you used to create that project.